src/overcooked_demo/server/static/js/index.js

```diff
@@ -0,0 +1,248 @@
+// Persistent network connection that will be used to transmit real-time data
```

Comment:
Is this still used?

Reply:
What do you mean?

Reply:
Oh sorry, I meant to say "is this whole file still used?"

Reply:
This one is still in use. This is actually where all the logic of the landing page is.